contract/src/offer-up.contract.js

```diff
@@ -139,8 +145,8 @@ export const start = async zcf => {
   const makeTradeInvitation = () =>
     zcf.makeInvitation(tradeHandler, 'buy items', undefined, proposalShape);
 
-  // Mark the publicFacet Far, i.e. reachable from outside the contract
-  const publicFacet = Far('Items Public Facet', {
+  // Use zone.exo to make a publicFacet suitable for use by remote callers.
+  const publicFacet = zone.exo('Items Public Facet', undefined, {
     makeTradeInvitation,
```

Comment:
I suggest inlining `makeTradeInvitation` here.

 1. That's the more usual idiom.
 2. I think exo methods have to use concise method syntax.

Reply:
> I think exo methods have to use concise method syntax.

"have to" is a bit too strong, but is directionally right. And exo singleton such as this one does not need to refer to `this`. Therefore, arrow functions happen to work. But they are not idiomatic and should not be. Methods of exo classes and exo class kits generally do need to refer to their `this`, and so must use concise method syntax. Also, methods in JavaScript class syntax are necessarily in concise method syntax. So for uniformity, I strongly recommend that even for exo singletons, methods use only concise method syntax.